releases/20250513-core-devs.md

```diff
@@ -62,3 +64,55 @@ The data will be served as part of a network presentation view on the World Expl
 
 ---
 
+### (2) Add Change Delegation to Helium Sub Daos
+
+Add change delegation to Helium Sub Daos.
+
+#### Motivation
+
+In order to change delegation currently, a staker must (1) claim all outstanding rewards, (2) undelegate (forfeiting the days rewards), (3) delegate to the new subnetwork. This is a UX pain point, and can be solved by a change delegation endpoint.
+
+#### Implementation
+
+The new endpoint internally undelegates then redelegated while preserving rewards state.
+
+#### Alternatives Considered
+
+None
+
+#### Impact
+
+* The core team does not expect any impact
+
+#### Drawbacks
+
+* None
+
+---
+
+### (3) Add Burn endpoint to Circuit Breakers
+
+Add a burn endpoint to the Circuit Breakers.
+
+#### Motivation
+
+In order to automate delegation rewards claims, the Helium Sub Daos smart contract needs a way to burn HNT for positions that have not voted. The current mechanism of burning requires the position holder to sign after the rewards have been transferred. This change will
+allow the contract to burn the HNT instead of transferring it, saving CUs and allowing tuktuk automation.
```

Comment:
```suggestion
In order to automate delegation rewards claims, the Helium Sub Daos on-chain program needs a way to burn HNT for positions that have not voted (as defined in HIP-141) . The current mechanism of burning requires the position holder to sign after the rewards have been transferred. This change will allow the contract to burn the HNT instead of transferring it, saving compute units and allowing automation through systems like tuktuk.
```